### ISSUE TITLE
do u have some contrastive datas between Albert with other pretraining models?

### ISSUE THREAD

**Issue by policeme** [google-research/ALBERT#54]

**Comment by Danny-Google** [comment]
Yes, you can find the comparison in the Chinese CLUE page (https://github.com/CLUEbenchmark/CLUE). Maybe it is because the way I trained it. The xxlarge model is sensitive to the downstream hyperparamters. You may need to run a search on the dev set to get good results.

**Comment by policeme** [comment]
but in this website, the xxlarge or base model was trained by bright, which one is yours

**Comment by Danny-Google** [comment]
The xxlarge model of the first and second tables are trained by me. The xxlarge model is not very stable as there were some problems in training the xxlarge model. I am still fixing that, but others should be fine.
